src/jdbc/Main.java

```diff
@@ -0,0 +1,32 @@
+package src.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+public class Main {
+    public static void main(String[] args) {
+        try (Connection connection = ConnectionManager.getConnection()){
+            Statement statement = connection.createStatement();
```

Comment:
Best practice is using PrepatedStatent over Statement

```
private static final String CONTACT_INSERT_SQL = "insert into contact (contact, contact_type) VALUES (?, ?)";

after that, using preparedStatemnt and it's method -- setString/set..(otherType) set values on it's order and execute a query
```

https://www.javatpoint.com/PreparedStatement-interface